source/deploy/deploy-mobile-apps-using-emm-provider.rst

```diff
@@ -17,13 +17,13 @@ Manage App Configuration Using AppConfig
 
 AppConfig is our recommended approach for app configuration and management. It was introduced by the `AppConfig Community <https://www.appconfig.org/about/>`__, a group of leading EMM providers and app developers who have come together to make it easier for developers and customers to drive mobility in business. 
 
-AppConfig provides an easy way to configure enterprise mobile apps with any of the EMM providers listed on the `AppConfig website <https://www.appconfig.org/members/>`__. Using AppConfig, you can manage default settings and security controls on both the public app stores, and custom-built mobile clients. For example, you can pre-configure your Mattermost server URL and username.
+AppConfig provides an easy way to configure mobile enterprise apps with any EMM providers listed on the `AppConfig website <https://www.appconfig.org/members/>`__. Using AppConfig, you can manage default settings and security controls on public app stores and custom-built mobile clients. For example, you can pre-configure your Mattermost server URL and username.
```

Comment:
```suggestion
AppConfig provides an easy way to configure enterprise mobile apps with any EMM providers listed on the `AppConfig website <https://www.appconfig.org/members/>`__. Using AppConfig, you can manage default settings and security controls on public app stores and custom-built mobile clients. For example, you can pre-configure your Mattermost server URL and username.
```

Reply:
I'm inclined to retain "enterprise mobile" because it refers to enterprise-level mobile apps vs. mobile apps for your enterprise. The former is more general while the latter leans more toward "your specific enterprise" which isn't aligned with the intention of the doc. :)

Reply:
Got it.